src/System/Console/ANSI/Windows/Emulator.hs

```diff
@@ -23,7 +23,11 @@ import Data.Colour (Colour)
 import Data.Colour.Names (black, blue, cyan, green, grey, lime, magenta, maroon,
   navy, olive, purple, red, silver, teal, white, yellow)
 import Data.Colour.SRGB (RGB (..), toSRGB)
+#ifdef MIN_VERSION_mintty
```

Comment:
No doubt this is correct ... but I failed to identify where this form of the macro is actually documented.

Reply:
Ignore that comment, I now understand C macros better.